.travis.yml

```diff
@@ -2,32 +2,28 @@ language: java
 sudo: false
 matrix:
   include:
-    - os: linux
-      dist: precise
+    - name: 'OpenJDK 7'
       jdk: openjdk7
-    - os: linux
+    - name: 'OpenJDK 8'
+      jdk: openjdk8
+    - name: 'Oracle JDK 8'
       jdk: oraclejdk8
-after_script:
-- "./gradlew build"
-- "./scripts/s3upload.sh"
-env:
-  global:
-  - S3_POLICY: ewogICJleHBpcmF0aW9uIjogIjIxMDAtMDEtMDFUMTI6MDA6MDAuMDAwWiIsCiAgImNvbmRpdGlvbnMiOiBbCiAgICB7ImFjbCI6ICJwdWJsaWMtcmVhZCIgfSwKICAgIHsiYnVja2V0IjogInNlbmRncmlkLW9wZW4tc291cmNlIiB9LAogICAgWyJzdGFydHMtd2l0aCIsICIka2V5IiwgInNtdHBhcGktamF2YS8iXSwKICAgIFsiY29udGVudC1sZW5ndGgtcmFuZ2UiLCAyMDQ4LCAyNjg0MzU0NTZdLAogICAgWyJlcSIsICIkQ29udGVudC1UeXBlIiwgImFwcGxpY2F0aW9uL3ppcCJdCiAgXQp9Cg==
-  - secure: mmr0NIQtQNnzMXPDN4YRBh8/NZX5FuPYlcrD/iZGMtEYKlXcmKziCzVu/XAvMGYPaeM0TCMSreWX2cD6KfFfcXQOtEoQGjngqHYWYjO9H3FwkUKJhjAEPqkAiI/kc5A9RMoM66eQvEcQ03mwyrXz3Xsuy826mKBUyMBcIvX0D/I=
-  - secure: aqXQn+NgNXEXL1GgzZP2+pqr0SVffAo6UZs4Tdu1l3eOVUhddAXpM3wZDqQpOMF1UfVCyIOymYTertVwFg9STMnnJHVy/lcRkG2E1w0bcs2S/goZtWGpr94EvIhm3ESw0rszkZLKgvbDnGQRd5/rqgRAqNq/aJnPnmocP+YfsQ0=
-  - secure: Z0ivY2rTHZaD/30+2Ta4UrFIjSro6epw+w3Ufu74YzTT3UcTIIb8zhNoMlcwZjTEtJu1IkDQljR3kYg3/1Gx3qZUwo98eq7TVVh+ozzQTcWu8lKz4+vIduS04mZTAZgEDuntkP86yGOfeRRezlG9LYuWmiQfMlOR4wZvjfYjPAk=
-notifications:
-  hipchat:
-    rooms:
-      secure: ko9f34ZvoDhf685wpxViPnWuNzaUYcQ3CBwdIsYWzGASMtdRY9X9pJWGwo/IfZIYoxrOFtCRWQEl9O0vq/DglB7T/uXsTyCvJurVg2d/Q5JLEgG1LQnW2vQGEeLbahiW9F/IT3htsfs+ZXRavKM2Bbokx1f9CLokehL+p0Z9xus=
-    template:
-    - '<a href="https://travis-ci.org/%{repository}/builds/%{build_id}">%{repository}
-      Build %{build_number}</a> on branch <i>%{branch}</i> by %{author}: <strong>%{message}</strong>
-      <a href="https://github.com/%{repository}/commits/%{commit}">View on GitHub</a>'
-    format: html
-    notify: true
-before_install:
-  - cat /etc/hosts # optionally check the content *before*
-  - sudo hostname "$(hostname | cut -c1-63)"
-  - sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts | sudo tee /etc/hosts
-  - cat /etc/hosts # optionally check the content *after*
+    - jdk: openjdk11
+      name: 'OpenJDK 11'
+  allow_failures:
+    - name: 'Oracle JDK 8'
+    - name: 'OpenJDK 7'
+    - name: 'OpenJDK 11'
+before_script:
```

Comment:
prism is only needed for the main helper libraries (sendgrid-java, sendgrid-php, etc..)